_articles/dual-booting.md

```diff
@@ -13,50 +13,13 @@ section: pop-ubuntu
 
 ---
 
-To dual boot Pop!\_OS alongside another OS, use the "Customize Partitions…" option when selecting a drive.
+### If Pop!\_OS is installed first
 
```

Comment:
If Pop!\_OS is not installed, follow the steps outlined in [INSTALL POP ARTICLE HYPERLINK]